src/mm/allocator/bump.rs

```diff
@@ -0,0 +1,46 @@
+//! A bump allocator.
```

Comment:
Not so much information please...

```suggestion
//! A bump allocator.
//!
//! A bump allocator is a very simple allocator design which has the drawback, that it can't deallocate memory.
```

Reply:
Added 👍